src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -1803,13 +1823,13 @@ def test_dyninvoke_uniq_declns_intent_cma_ops():
         .unique_declns_by_intent("gh_columnwise_operator")
     args_inout = [arg.declaration_name for arg in args['inout']]
     assert args_inout == ['cma_opc']
+    # 'cma_op1' is "write" arg in columnwise_op_mul_kernel_type call
     args_out = [arg.declaration_name for arg in args['out']]
     assert args_out == ['cma_op1']
+    # 'cma_op1' is "read" arg in columnwise_op_app_kernel_type call
     args_in = [arg.declaration_name for arg in args['in']]
     assert args_in == ['cma_op1', 'cma_opb']
 
-    assert LFRicBuild(tmpdir).code_compiles(psy)
-
```

Comment:
Does this code not compile? If not, please could you reinstate this but commented out with an explanation.

Reply:
This compilation line is for `test_dyninvoke_uniq_declns_intent_ops(tmpdir):` test. `test_dyninvoke_uniq_declns_intent_cma_ops():` did not have compilation as there is already a compilation check for "20.5_multi_cma_invoke.f90" in the `dynamo0p3_cma_test.py`. I added it here, though, to be on the safe side.